libs/hdf-converters/src/neuvector-mapper.ts

```diff
@@ -0,0 +1,311 @@
+    return (moduleName: string) => {
+      return this.rawData.report.modules?.find(
+        (value: RESTScanModule) => value.name === moduleName
+      );
+    };
+  }
+
+  mappings: MappedTransform<
+    ExecJSON.Execution & {passthrough: unknown},
+    ILookupPath
+  > = {
+    platform: {
+      name: 'Heimdall Tools',
+      release: HeimdallToolsVersion,
+      target_id: null
+    },
+    version: HeimdallToolsVersion,
+    statistics: {
+      duration: null
+    },
+    profiles: [
+      {
+        name: 'Neuvector',
+        title: null,
+        version: null,
+        maintainer: null,
+        summary: null,
+        license: null,
+        copyright: null,
+        copyright_email: null,
+        supports: [],
+        attributes: [],
+        depends: [],
+        groups: [],
+        status: 'loaded',
+        controls: [
+          {
+            path: 'report.vulnerabilities',
+            key: 'id',
+            tags: {
+              cves: {path: 'cves'},
+              cpes: {
+                path: 'cpes',
+                transformer: (cpes: string[]) => (!cpes ? undefined : cpes)
+              },
+              cwe: {
+                path: 'description',
+                transformer: cweTags
+              },
+              nist: {
+                path: 'description',
+                transformer: (description: string) =>
+                  nistTags(cweTags(description))
+              },
+              ghsa: {path: 'name', transformer: ghsaTag},
+              rhsa: {path: 'name', transformer: rhsaTag},
+              // `score` is confirmed to be CVSS v2 in https://github.com/neuvector/scanner/blob/765fb1db2cf678ea6c6d386f3eb0f720311d745a/cvetools/cvesearch.go#L1416
+              score: {path: 'score'},
+              vectors: {path: 'vectors'},
+              // In the neuvector/scanner dashboard, the CVSS v2 and v3 scores are selectable by a dropdown.
+              vectors_v3: {path: 'vectors_v3'},
+              score_v3: {path: 'score_v3'},
+              // Appears to be CVSS v3, since info for v2 and v4 doesn't always exist for CVEs.
+              severity: {path: 'severity'},
+              source: {
+                path: 'package_name',
+                transformer: (packageName: string) =>
+                  this.getModule(packageName)?.source
+              },
+              status: {
+                path: 'name',
+                transformer: (name: string) =>
+                  this.rawData.report.modules
+                    ?.find((module) => module.cves?.find(cveIdMatches(name)))
+                    ?.cves?.find(cveIdMatches(name))?.status
+              },
+              feed_rating: {path: 'feed_rating'},
+              link: {path: 'link'},
+              // Both fields are Unix epoch timestamps, in seconds.
+              published_timestamp: {path: 'published_timestamp'},
+              last_modified_timestamp: {path: 'last_modified_timestamp'},
+              ...universalTags()
```

Comment:
i don't see why they need to be split out into a separate function.  just leave them here.